ts/att/timeSpan.ts

```diff
@@ -0,0 +1,34 @@
+export class TimeSpan {
+
+    public totalSeconds: number;
+
+    public constructor(hours: number, minutes: number, seconds: number) {
+        if (hours < 0 || minutes < 0 || seconds < 0) {
+            throw RangeError;
+        }
+        this.totalSeconds = seconds + (hours * 60 + minutes) * 60;
+    }
+
+    public hours(): number {
```

Comment:
Please write test for this method.

Reply:
For you information, all method must be at least backed up by ONE unit test.

Reply:
Moreover, unit test also serves as **documentation** . It means if people don't understand what this particular method is doing, they can look at its test code to understand about its functionality.

Reply:
Done